lib/compilers/lfortran.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2022, Compiler Explorer Authors
```

Comment:
Please update the copyright year to 2024

```suggestion
// Copyright (c) 2024, Compiler Explorer Authors
```

Reply:
Done